app/src/test/java/nerd/tuxmobil/fahrplan/congress/sharing/SimpleSessionFormatTest.kt

```diff
@@ -26,42 +26,44 @@ class SimpleSessionFormatTest {
     private val systemTimezone = TimeZone.getDefault()
     private val systemLocale = Locale.getDefault()
 
-    private val session1 = Session("S1").apply {
-        title = "A talk which changes your life"
-        roomName = "Yellow pavilion"
-        dateText = "2019-12-27T11:00:00+01:00"
-        dateUTC = DateParser.parseDateTime(dateText)
-        url = "https://example.com/2019/LD3FX9.html"
-        slug = "LD3FX9"
-    }
-
-    private val session2 = Session("S2").apply {
-        title = "The most boring workshop ever"
-        roomName = "Dark cellar"
-        dateText = "2019-12-28T17:00:00+01:00"
-        dateUTC = DateParser.parseDateTime(dateText)
-        url = "https://example.com/2019/U28VSA.html"
-        slug = "U28VSA"
-    }
-
-    private val session3 = Session("S3").apply {
-        title = "Angel shifts planning"
-        roomName = "Main hall"
-        dateText = "2019-12-29T09:00:00+01:00"
-        dateUTC = DateParser.parseDateTime(dateText)
-        links = "https://events.ccc.de/congress/2019/wiki/index.php/Session:A/V_Angel_Meeting"
-        url = "https://example.com/2019/U28VSA.html"
-        slug = "U28VSA"
-    }
-
-    private val session4 = Session("S4").apply {
-        title = "Central european summer time"
-        roomName = "Sunshine tent"
-        dateText = "2019-09-01T16:00:00+02:00"
-        dateUTC = DateParser.parseDateTime(dateText)
-        url = "https://example.com/2019/U9SD23.html"
-        slug = "U9SD23"
-    }
+    private val session1 = Session(sessionId = "S1",
+        title = "A talk which changes your life",
+        roomName = "Yellow pavilion",
+        dateText = "2019-12-27T11:00:00+01:00",
+        dateUTC = DateParser.parseDateTime("2019-12-27T11:00:00+01:00"),
+        url = "https://example.com/2019/LD3FX9.html",
+        slug = "LD3FX9",
+        )
+
+    private val session2 = Session(sessionId = "S2",
+        title = "The most boring workshop ever",
+        roomName = "Dark cellar",
+        dateText = "2019-12-28T17:00:00+01:00",
+        dateUTC = DateParser.parseDateTime("2019-12-28T17:00:00+01:00"),
+        url = "https://example.com/2019/U28VSA.html",
+        slug = "U28VSA",
+            )
```

Comment:
Inconsistent indentation.

Have you considered automatically enforcing a code style? We're using [spotless](https://github.com/diffplug/spotless) (which uses ktlint for Kotlin files).

Reply:
:heavy_check_mark: Applied formatting.

Yes, but I haven't taken the time to look into it.